.github/workflows/ubuntu-20.04.yml

```diff
@@ -0,0 +1,212 @@
+---
+name: ubuntu-20.04
```

Comment:
Copy file for ubuntu18.04. adapt compiler versions.
I think I prefer having twice almost the same file, than having twice the same content in a single file. 
If there is a nice way to define different compiler/python versions for two runs-on, this could be a solution, but for now, I would keep it as it is.

Reply:
I think we should have a single workflow file for both Ubuntu versions. Otherwise, it will be tedious to keep them in sync. GitHub Actions are quite flexible: e.g., you can execute part of the steps only for specific OS-compiler pairs.

Reply:
Ubuntu is now in a single file with exclusion.